spec/views/layouts/header.html.erb_spec.rb

```diff
@@ -30,6 +30,11 @@ def true_user
       expect(rendered).to match CGI.escapeHTML user.display_name
       expect(rendered).to match CGI.escapeHTML user.email
     end
+
+    it "renders help issue link on the header" do
```

Comment:
Super minor suggestion, this wording is definitely accurate, as I read it I thought maybe it’d be helpful to be slightly more specific and say that the help link should be in the menu. 

Again super minor suggestion…take it or leave it ☺️

```suggestion
    it "renders help issue link in the drop down menu" do
```

Reply:
Looks good!

Reply:
> Super minor suggestion, this wording is definitely accurate, as I read it I thought maybe it’d be helpful to be slightly more specific and say that the help link should be in the menu.
> 
> Again super minor suggestion…take it or leave it ☺️

Thanks @mattzollinhofer for the suggestion. It indeed is good point but PR is already merged so don't think it makes sense to open another PR for this small change.